platform/qt/README.md

```diff
@@ -0,0 +1,68 @@
+# Mapbox Qt SDK
+
+[![Travis](https://travis-ci.org/mapbox/mapbox-gl-native.svg?branch=master)](https://travis-ci.org/mapbox/mapbox-gl-native/builds)
+
+Provides [Qt](http://www.qt.io/) example applications and APIs via `QMapboxGL`
+and `QQuickMapboxGL`:
+
+**QMapboxGL** - [QObject](http://doc.qt.io/qt-5/qobject.html)-based, implements
```

Comment:
I would simply say that is low-level instead of QObject-based because almost everything on Qt is QObject-based.